docs/ja/ai-integration/overview.md

```
@@ -147,12 +149,17 @@ GeonicDB は [Model Context Protocol (MCP)](https://modelcontextprotocol.io/) 
+- **トランスポート**: Streamable HTTP (JSON レスポンスモード)
 - **プロトコルバージョン**: 2025-03-26
 - **動作モード**: ステートレス(Lambda 互換)
-- **認証**: `AUTH_ENABLED=true` の場合、JWT Bearer トークンによるアクセス制御とテナント分離が適用されます### Claude Desktop の設定
+- **認証**: `AUTH_ENABLED=true` の場合、JWT Bearer トークンによるアクセス制御とテナント分離が適用されます
+
+### Claude Desktop の設定
+
+#
 
-#### ローカル開発環境 (認証なし)
+### ローカル開発環境 (認証なし)
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)